src/index.js

```diff
@@ -528,7 +528,12 @@ export default (Bookshelf, options = {}) => {
                             // Remove all but the last table name, need to get number of dots
                             column = internals.formatRelation(internals.formatColumnNames([column])[0]);
                             if (!_isArray(value)) {
-                                value = split(String(value), { keepQuotes: true, sep: ',' });
+                                if (value){
```

Comment:
This will break for values such as `false`. Check if it equal null else do the split.